backend/qemu.pm

```diff
@@ -931,7 +931,12 @@ sub start_qemu ($self) {
 
         if ($vars->{VNC}) {
             my $vncport = $vars->{VNC} !~ /:/ ? ":$vars->{VNC}" : $vars->{VNC};
-            sp('vnc', [qv "$vncport share=force-shared"]);
+            my $extravars = $vars->{VNC_EXTRA_VARS};
+            if($extravars) {
+                sp('vnc', [qv "$vncport share=force-shared $extravars"]);
+            } else {
+                sp('vnc', [qv "$vncport share=force-shared"]);
+            }
```

Comment:
I'd avoid the repetition and use `defined` because someone might for some reason want to append `0`.

```suggestion
            my $extravars = $vars->{VNC_EXTRA_VARS};
            $extravars = defined $extravars ? " $extravars" : '';
            sp('vnc', [qv "$vncport share=force-shared$extravars"]);
```

Reply:
Then I guess it can be as simple as:

```suggestion
            my $extravars = $vars->{VNC_EXTRA_VARS} // '';
            sp('vnc', [qv "$vncport share=force-shared$extravars"]);
```

Reply:
I took @Martchus suggestion. @okurz suggestion is missing a space AFAICT.

Reply:
You are right